### DISCUSSION TITLE
Security vulnerability issue

### DISCUSSION THREAD

**Discussion by codeMonkey-shin** [coder/code-server#6864] [General]
![image](https://github.com/coder/code-server/assets/80636401/82886511-1585-41ae-a13a-1c7856b1f51e)

My harbor says that there is a vulnerability in the Docker image of codercom/code-server. I wonder if there is any problem with this

**Comment by code-asher** [comment]
We can update the Docker container image if we need to, but according to the image above it looks like there are no fixed versions yet?
